src/librustc_infer/infer/error_reporting/mod.rs

```diff
@@ -824,10 +824,13 @@ impl<'a, 'tcx> InferCtxt<'a, 'tcx> {
             .rev()
             .filter_map(|param| match param.kind {
                 ty::GenericParamDefKind::Lifetime => None,
-                ty::GenericParamDefKind::Type { has_default, .. } => {
+
+                ty::GenericParamDefKind::Type { has_default, .. } |
+                // FIXME(const_generics:defaults)
```

Comment:
isn't that fixme outdated now?

Reply:
To be honest no idea what these FIXME's exactly were waiting on so I left them